File: system/framework/shops/silpo/ItemModel.php
<?php

namespace framework\shops\silpo;

class ItemModel{
    public string $label;
    public string $image;
    public int $category;
    public string $brand;
    public string $packege;
    public int $units;
    public int $term;
    public ?array $barcodes;
    public ?array $consist;
    public ?float $calorie;
    public ?float $carbohydrates;
    public ?float $fat;
    public ?float $proteins;
    public $additional;

    public function __construct(
        ?int $id = null,
        ?string $label = null,
        ?string $image = null,
        ?int $category = null,
        ?int $brand = null,
        ?int $packege = null,
        ?int $units = null,
        ?int $term = null,
        ?string $barcodes = null,
        ?string $consist = null,
        ?float $calorie = null,
        ?float $carbohydrates = null,
        ?float $fat = null,
        ?float $proteins = null,
        $additional = null
    ) {
        $this->id = $id;
        $this->label = $label;
        $this->image = $image;
        $this->category = $category;
        $this->brand = $brand;
        $this->packege = $packege;
        $this->units = $units;
        $this->term = $term;
        $this->barcodes = $this->stringConvert($barcodes);
        $this->consist = $this->stringConvert($consist);
        $this->calorie = $calorie;
        $this->carbohydrates = $carbohydrates;
        $this->fat = $fat;
        $this->proteins = $proteins;
        $this->additional = json_decode($additional);
    }
}

File: priceapp.API/Models/ItemModel.cs
namespace priceapp.API.Models;

public class ItemModel
{
    public int Id { get; set; }
    public string Label { get; set; }
    public string Image { get; set; }
    public int Category { get; set; }
    public int Brand { get; set; }
    public int Package { get; set; }
    public double Units { get; set; }
    public double Term { get; set; }
    //public int[]? Consist { get; set; }
    public double? Calorie { get; set; }
    public double? Carbohydrates { get; set; }
    public double? Fat { get; set; }
    public double? Proteins { get; set; }
    public object? Additional { get; set; }
}